CODEOWNERS

```diff
@@ -92,6 +92,8 @@ extensions/filters/common/original_src @snowp @klarose
 /*/extensions/filters/network/wasm @PiotrSikora @lizan
 # webassembly common extension
 /*/extensions/common/wasm @PiotrSikora @lizan
+# webassembly runtimes
+/*/extensions/wasm_runtime/ @PiotrSikora @lizan
```

Comment:
Any reason for not putting it under `extensions/common/wasm/runtimes` or similar?

Reply:
I don't think we put any actual extension registrations under `extensions/common`, `extensions/common` is for libraries used by extensions.